docs/design/proposals/intree-migrate.md

```diff
@@ -0,0 +1,106 @@
+# In-tree storage plugin to CSI Driver Migration
+
+Prior to CSI, Kubernetes provided a powerful volume plugin system. These volume
```

Comment:
except `kubernetes.io/rbd` everything looks to be a generic one instead of mentioning the same content can we link to the kube design doc https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md?